src/App.tsx

```diff
@@ -1,7 +1,13 @@
 import React from 'react';
 
-export class App extends React.Component {
-  state = {
+type Props = {};
+
+interface State {
+  pressedKey: string | null;
+}
+
+export class App extends React.Component<Props, State> {
+  state: Readonly<State> = {
```

Comment:
TS will infer it here from initial object, you can safely go with `state = {`

Reply:
I think Volodymyr told us to use `state: Readonly<State> = {}` because it provides a higher level of safety than just `state = {}`.

Reply:
Yes, my bad here